src/plugin.js

```diff
@@ -157,4 +159,48 @@
     isArray: Array.isArray
   };
 
+  // Polyfill the tech function with properties for the current tech. This is
+  // a mostly-complete implementation of the old tech behavior.
+  Player.prototype.loadTech_ = function() {
+    var player = this;
+
+    originals.Player.loadTech_.apply(player, arguments);
+
+    var tech = this.tech_;
+    var ownKeys = Object.keys(tech);
+    var proto = tech.constructor.prototype;
+    var protoKeys = Object.keys(proto);
+
+    this.polyfilledTechKeys_ = ownKeys.concat(protoKeys.filter(function(key) {
+
+      // Only take protoKeys that are NOT found in the ownKeys.
+      return ownKeys.indexOf(key) === -1;
```

Comment:
How would that work? `tech[key]` would match properties both on the tech itself and up the prototype chain. This way it's explicit.